File: QRok/Models/SurveyOption.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace QRok.Models
{
    public class SurveyOption
    {
        public int SurveyId { get; set; }
        public Survey Survey { get; set; }

        public int OptionNumber { get; set; }

        [Required]
        [MaxLength(255)]
        public string Title { get; set; }

        public int Count { get; set; }

    }
}

File: QRok/wwwroot/js/KOViewModels/SurveyOption.js
function SurveyOption(obj) {
	this.SurveyId = ko.observable(0);
	this.OptionNumber = ko.observable(0);
	this.Title = ko.observable("");
	this.Count = ko.observable(0);

	if (obj != null) {
		this.SurveyId(obj.SurveyId);
		this.OptionNumber(obj.OptionNumber);
		this.Title(obj.Title);
		this.Count(obj.Count);

	}
}